html/js/melpa.js

```diff
@@ -150,7 +151,7 @@
       pkgs.push(new melpa.Package({
         name: name,
         version: version,
-        commit: built.props.commit,
+        commit: props.commit || '',
```

Comment:
> Or, rather, the default value of '' looks undesirable. :-)

@purcell got it :-) I think it's a matter of preference since JavaScript is not a typed language. `commit` property is assumed to be a string then my thinking is that the default value should be a string too. If we keep that aside then `''`, `null`, `undefined` all of them are equal as a boolean value and `props.commit` expression will yield `undefined`.